### ISSUE TITLE
Small little point next to lines - Is this expected ?

### ISSUE THREAD

**Issue by steinemann** [Reviewable/Reviewable#888]
Hi,

I can see small little black points next to each line on the diff view.
They appear w/o custom style settings

Is this somehow expected ?

![image](https://user-images.githubusercontent.com/5727544/145022995-1575f6d3-2ce2-48d1-9a1b-1146c77366b9.png)

Thanks,
Peter

**Comment by pkaminski** [comment]
No, that doesn't look right, but then neither do the line numbers -- Reviewable doesn't have an option to show those.  Are you _sure_ you don't have a custom stylesheet applied?

**Comment by zware** [comment]
On Firefox, I was seeing the dots recently with no custom styling, but they seem to be gone at the moment.  I don't recall if line numbers were also shown (but that would be a nice feature :)).

**Comment by steinemann** [comment]
For the line numbers I use the stylesheet that's proposed in the official documentation. 

But I also can see them without the stylesheet.
I use Chrome as Browser.

**Comment by pkaminski** [comment]
This doesn't repro for me, either with or without the stylesheet, in Chrome on Windows.  What is the exact version of Chrome and platform you're running on?  Can you also provide screenshots of the diff view without the line numbers, and of your account settings panel for the record?  This looks like Reviewable computing the wrong line height for some reason but it's gonna be tricky to run down.  Thanks.

**Comment by steinemann** [comment]
I am using Chrome 96.0.4664.45 on Windows 10 (x64)
I am using a Chrome view resize of 125 % percent

Here is a snippet without the **custom syntax:**

![image](https://user-images.githubusercontent.com/5727544/145542659-a36f05ab-b3e5-4ec4-ada8-a3864108624d.png)

And a screenshot of my **Account settings:**

![image](https://user-images.githubusercontent.com/5727544/145542934-ce9e3cf3-158e-45e0-be96-4cf24a92b311.png)

I could figure out that it's part of this `div`: ( inside Chrome )

```
<div class="diff line keep ">
  <div class="line gutter" data-line-ids="left-143 right-143"></div>
  <div class="line wrapper diffwidth  " data-line-number="143">
    <div class="hljs">
      <div class="monospace line content diffwidth    ">                    steps {</div>
    </div>
    <div class="left line discussions diffwidth" id="diff-5-Mohgj7knRPkOGk3Zc42-left-143">
    </div>
    <div class="right line discussions diffwidth" id="diff-5-Mohgj7knRPkOGk3Zc42-right-143">
    </div>
  </div>
</div>
```

**Comment by pkaminski** [comment]
The mystery deepens...  I'm running the same version of Chrome on Windows.  I set my font size to 100%, and tried Chrome scaling from 100% to 150%.  I even tried putting the window both on my 100% scaled and 175% scaled monitors.  Nothing managed to reproduce the issue.

Can you play around with various values to see what's causing it to break for you?  Try putting the font back to its default 89% size, make sure Chrome scaling is at 100%, put it on a 100% scale monitor, etc.

Or if you feel comfortable poking around the DOM, the basic idea is that Reviewable measures the height of a line of text in the chosen monospace font and sets it on `.line.wrapper` as `line-height`.  Then the `.line.content` has a `linear-gradient` that is transparent for the first line height pixels, but goes to `#666` thereafter to indicate a wrapped line.  So either the line height measurement is wrong, or the line ends up taller than expected for some reason, and ends up showing a bit of the overflow gradient...

Thanks!

**Comment by pkaminski** [comment]
Ping!  We'll need more information from you @steinemann to get this mystery solved.  Thanks!

**Comment by steinemann** [comment]
Hi Piotr,

Thanks for pinging.

So finally, I have good and bad news.

__The bad news:__
I tried to figure out the root cause and played around as you suggest, but was not able to figure out the root cause or a reproduction scenario.

__The good news:__
The issue disappeared after we upgraded our Reviewable version to v3359.5145
I also tried now to re-create the issues on this version, but failed.

So I finally guess that it was fixed with another change.

Thanks
Peter
